PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/actions/WebDeployAction.java

```diff
@@ -62,50 +63,38 @@ public void actionPerformed(AnActionEvent event) {
             return;
         }
 
-        ApplicationManager.getApplication().invokeLater(() -> {
-            runConfiguration(project);
-        });
+        ApplicationManager.getApplication().invokeLater(() -> runConfiguration(project));
     }
 
     private void runConfiguration(Project project) {
-        final RunManagerImpl manager = RunManagerImpl.getInstanceImpl(project);
```

Comment:
This part belongs to our new feature. We should not user impl class, so change to the wrapper.